recipe/meta.yaml

```diff
@@ -10,33 +10,21 @@ source:
 
 build:
   noarch: python
```

Comment:
The package probably cannot be noarch now, right? Will we need post-link.bat?

Reply:
julia-feedstock is only available on macOS and linux so I'm assuming we can leave it as noarch until that changes?

Reply:
It is better the opposite in that case b/c these noarch packages will be available when the Julia package lands on those platforms, creating possible solutions but faulty installations.